src/content/docs/accounts/accounts-billing/account-structure/new-relic-account-structure.mdx

```diff
@@ -1,34 +1,43 @@
   - /docs/accounts/accounts-billing/new-relic-one-pricing-users/account-user-structure
   - /docs/accounts/accounts-billing/new-relic-one-pricing-users/new-relic-account-structure
 ---
 
-Depending on your account/user model, you have different options for adding and managing accounts and assigning users to them. 
+Depending on your account/user model, you have different options for adding and managing accounts and assigning users to them. We have two models: 
 
-## Impacts of account/user model [#user-model]
+* [New Relic One account/user model](#new-model)
+* [Original account/user model](/docs/accounts/original-accounts-billing/original-users-roles/original-account-structure)
 
-How you manage accounts and users differs based on which of our account/user models your organization is on. To learn more about the account/user models and determine which you're on, see [User models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models). The major differences are: 
+<Callout variant="important">
+Note that the account/user model is not directly related to our [two pricing plans](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#pricing-plans). 
+</Callout>
 
-* Original account/user model: Our original model used a master/sub-account structure, where sub-accounts inherited settings from master accounts. If you're on this model, see the [Original user model docs](/docs/accounts/accounts-billing/account-structure/mastersub-account-structure). 
-* New Relic One account/user model: This is our new, improved model. We currently are migrating some customer accounts to this model and eventually all our customers will be on this model. To learn more and see if you can migrate your users, see [Migrate users](/docs/accounts/original-accounts-billing/original-users-roles/user-migration). To learn more about how account and user management works, keep reading. 
+## New Relic One account/user model [#new-model]
 
-## Account structure for New Relic One account/user model [#account-structure]
+<Callout variant="important">
+This section is about organizations on the New Relic One account/user model, not the original model. [Learn more about the difference.](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models) 
+</Callout>
 
-For organizations on the New Relic One account/user model, here are some details about how organization and account management work: 
+At New Relic, an "organization" is the representation of a New Relic customer. The organization contains everything pertinent to a New Relic customer: its accounts, its users, and its data. For organizations on [New Relic One pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/), usage that impacts billing is aggregated at the organization level. 
```

Comment:
I once used "pertinent" in a voicemail message and my college chums mocked me endlessly for it. Maybe "relevant" is better? Or even "about"?